edsdme/components/PartnerCardsStyles.js

```diff
@@ -840,6 +840,38 @@ export const partnerCardsStyles = css`
   .all-filters-wrapper-mobile .all-filters-footer-clear-btn-mobile:focus {
     border-color: ${blueColor};
   }
+  
+  .filter-info {
+    display: flex;
+    margin: 0 15px 5px 0;
```

Comment:
@SonjaPopovic one final small UI bug I found, I believe the info icon should align with the checkboxes according to the design?

<img width="292" alt="Screenshot 2024-11-19 at 13 17 33" src="https://github.com/user-attachments/assets/c838223e-ee5f-49a4-8395-cdd3061b002f">